### PR TITLE
RBEA-695: call/mail add alignment options

### PR DESCRIPTION
**Pull request by vibhavsalelkar**

### CODE REVIEW

**Review by Nayan-Gaikwad — APPROVED** [review]

File: src/blocks/call-mail-button/components/inspector.js [hunk 1]
```diff
@@ -8,23 +8,19 @@ const { Component, Fragment } = wp.element;
 import FontIconPicker from "@fonticonpicker/react-fonticonpicker";
 import ResponsiveBlocksIcon from "../../../ResponsiveBlocksIcon.json";
 import renderSVG from "../../../renderIcon";
-import fontOptions from "../../../utils/googlefonts";
 import { loadGoogleFont } from "../../../utils/font";
-import ResponsiveSpacingControl from "../../../settings-components/Responsive Spacing Settings";
 import TypographyHelperControl from "../../../settings-components/Typography Settings";
 import ResponsiveMarginControl from "../../../settings-components/Responsive Spacing Settings/Responsive Margin Control";
 import ResponsivePaddingControl from "../../../settings-components/Responsive Spacing Settings/Responsive Padding Control";
```
Comment (ratnesh-kadam) [inline src/blocks/call-mail-button/components/inspector.js]: @vibhavsalelkar @Nayan-Gaikwad why those file names has space?

**Review by ratnesh-kadam — COMMENTED** [review]
@vibhavsalelkar @Nayan-Gaikwad why some of the file names has space?

File: src/blocks/call-mail-button/components/inspector.js (hunk 1 above)
Comment (Nayan-Gaikwad) [reply src/blocks/call-mail-button/components/inspector.js]: @vibhavsalelkar Please remove spaces in the folder names for all setting-components  as we did for BorderSettingsControl

Comment (vibhavsalelkar) [reply src/blocks/call-mail-button/components/inspector.js]: @ratnesh-kadam @Nayan-Gaikwad removed spaces from folder names in recent commit.

**Review by Nayan-Gaikwad — APPROVED** [review]

**Review by ratnesh-kadam — APPROVED** [review]

**Pull request merged**